src/countries/index.test.ts

```diff
@@ -46,12 +46,22 @@ describe('country helpers', () => {
       (opts: { name: string; country: string }) => {
         expect(getConfigByISOName(opts.country)).toMatchSnapshot();
       },
-      Object.keys(prayersConfigs).map((country) => {
-        return {
-          name: country,
-          country,
-        };
-      })
+      [
+        ...Object.keys(prayersConfigs).map((country) => {
```

Comment:
Why this change was needed? Didnt previous block capture all countries?

Reply:
Check the snapshot. I think it now has dup snapshots

Reply:
Checking the snapshot. Should I add test inside the `describe('getConfigByISOName')` block to test the configurations for Netherlands and Switzerland?

Reply:
please read the logic and understand it. 


if you didn't understand here is explanation: 
it already iterates on each country. so you don't need to add a country. but test will fail bcoz of snapshot mismatch. so all you need to review the snapshot differences and accept the update if they seem to be correct.